### PR TITLE
[Spanish] localize 'Firewall'

### LINKED ISSUE #1375: [Spanish] localize 'Firewall'
Localize the `content/en/firewall.md` document to Spanish

### PR DESCRIPTION
**Pull request by tinniaru3005**
### Describe your changes
Converted `firewall.md` to Spanish.

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1375 
Closes #1375 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha, I cannot understand how the extra changes are getting added. Can you please guide me on how to avoid it and how I can I rectify it here?

**Comment by electrocucaracha** [comment]
> Hey @electrocucaracha, I cannot understand how the extra changes are getting added. Can you please guide me on how to avoid it and how I can I rectify it here?

It seems those changes are coming from the `main` branch, so when the PR is created targeting to `dev-es` it's when they appear.

**Comment by tinniaru3005** [comment]
Thanks @electrocucaracha for explaining me. Are the changes fine now?

File: content/es/firewall.md
Comment (electrocucaracha) [inline content/es/firewall.md:5]:
```suggestion
status: Completed
category: Tecnología
tags: ["security", "", ""]
```

File: content/es/firewall.md
Comment (electrocucaracha) [inline content/es/firewall.md:8]:
```suggestion
## ¿Qué es?
```

File: content/es/firewall.md
Comment (electrocucaracha) [inline content/es/firewall.md:20]:
```suggestion
## ¿Cómo ayuda?
```

File: content/es/firewall.md
Comment (electrocucaracha) [inline content/es/firewall.md:17]:
```suggestion
Por ejemplo, en una aplicación bancaria basada en [microservicios]((/microservices/), los servicios se comunican entre sí mediante la transmisión de datos financieros altamente confidenciales a través de su red.
```

File: content/es/firewall.md
Comment (electrocucaracha) [inline content/es/firewall.md:2]:
```suggestion
title: Cortafuegos
```

File: content/es/firewall.md
Comment (electrocucaracha) [inline content/es/firewall.md:22]:
```suggestion
Un cortafuegos examina el tráfico de la red utilizando reglas predefinidas.
```

File: content/es/.wordlist.txt
Comment (electrocucaracha) [inline content/es/.wordlist.txt:118]: After accepting the below suggestion, this won't be needed.
```suggestion
```

**Comment by electrocucaracha** [comment]
> Thanks @electrocucaracha for explaining me. Are the changes fine now?

Apparently, the spellchecker reports the `enrutamiento` and `microservices` words are missing in the list

**Comment by tinniaru3005** [comment]
> > Thanks @electrocucaracha for explaining me. Are the changes fine now?
> 
> Apparently, the spellchecker reports the `enrutamiento` and `microservices` words are missing in the list

Can you please guide me on how I can make those changes

**Comment by electrocucaracha** [comment]
> > > Thanks @electrocucaracha for explaining me. Are the changes fine now?
> > 
> > 
> > Apparently, the spellchecker reports the `enrutamiento` and `microservices` words are missing in the list
> 
> Can you please guide me on how I can make those changes

Sure, with the latest additions, it seems like `enrutamiento` is the only missing word. This word has to be added (in alphabetical order) in this [file](https://github.com/cncf/glossary/blob/dev-es/content/es/.wordlist.txt). If you use the terminal, you can execute the following bash instructions

```bash
$ git remote -v
origin  git@github.com:electrocucaracha/glossary.git (fetch)
origin  git@github.com:electrocucaracha/glossary.git (push)
upstream        https://github.com/cncf/glossary (fetch)
upstream        https://github.com/cncf/glossary (push)
$ git branch
* dev-es
  main
$ git pull upstream dev-es
$ gh pr checkout 1404
$ git rebase dev-es
```
You'll need to solve some rebase issues here before adding the missing word.
```bash
$ echo "enrutamiento" >> content/es/.wordlist.txt
$ sort -f -o content/es/.wordlist.txt content/es/.wordlist.txt
```

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha, is it fine now?

**Review by electrocucaracha — APPROVED** [review]

File: content/es/firewall.md
Comment (tinniaru3005) [reply content/es/firewall.md]: Ya, I did that :)

**Review by raelga — APPROVED** [review]

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha, Can the PR be merged now?

File: content/es/firewall.md
Comment (electrocucaracha) [inline content/es/firewall.md:2]:
```suggestion
title: Cortafuegos
```

File: content/es/.wordlist.txt
Comment (electrocucaracha) [inline content/es/.wordlist.txt:72]:
```suggestion
Engine
enrutamiento
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
I found two small changes which are needed before merge this.

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha. Is it good to merge now?

**Review by electrocucaracha — APPROVED** [review]

**Comment by electrocucaracha** [comment]
> Hey @electrocucaracha. Is it good to merge now?

It looks good to me, the `Squash and merge` button is disabled on my side. Maybe, we need to wait for @krol3 review.

File: content/es/firewall.md
Comment (krol3) [reply content/es/firewall.md]: @electrocucaracha  In my first review I suggested using the word `Firewall` instead of Cortafuegos. I think we discuss this in the meeting too.

File: content/es/firewall.md
Comment (electrocucaracha) [reply content/es/firewall.md:2]:
```suggestion
title: Firewall
```

**Comment by electrocucaracha** [comment]
@tinniaru3005 it seems like the suggestion that @krol3 made wasn't applied. Could you use `firewall` instead of `cortafuegos`? More likely, the word list will be affected.

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha is it fine now?

**Review by Mouly22 — APPROVED** [review]

**Comment by electrocucaracha** [comment]
@tinniaru3005 it seems that this PR has more not related changes, can you verify it and fix it?

**Comment by tinniaru3005** [comment]
Hey @electrocucaracha when I fork only the main branch gets forked. I am facing this issue in all the PRs. Can you please tell me how I can fork a particular branch.

**Comment by electrocucaracha** [comment]
> Hey @electrocucaracha when I fork only the main branch gets forked. I am facing this issue in all the PRs. Can you please tell me how I can fork a particular branch.

Your Spanish translation PRs have to be based on `dev-es` instead of `main`, there are some information [here](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#localization-team-guidelines)

**Comment by krol3** [comment]
> > Hey @electrocucaracha when I fork only the main branch gets forked. I am facing this issue in all the PRs. Can you please tell me how I can fork a particular branch.
> 
> Your Spanish translation PRs have to be based on `dev-es` instead of `main`, there are some information [here](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#localization-team-guidelines)

@tinniaru3005 Please review your PR, you need to create a branch from `dev-es` and add your changes. 
cc: @electrocucaracha

**Comment by krol3** [comment]
> > Hey @electrocucaracha when I fork only the main branch gets forked. I am facing this issue in all the PRs. Can you please tell me how I can fork a particular branch.
> 
> Your Spanish translation PRs have to be based on `dev-es` instead of `main`, there are some information [here](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#localization-team-guidelines)

@tinniaru3005 Please review your PR, you need to create a branch from `dev-es` and add your changes.

**Review by krol3 — CHANGES_REQUESTED** [review]
@tinniaru3005  You need to add your changes to the `dev-es` branch.

**Comment by krol3** [comment]
Hi @tinniaru3005 ! how are you? Could you resolve the conflicts? do you need help ?

**Pull request closed without merging**
